### PR TITLE
chore: bundlr update

### PR DESCRIPTION
**Pull request by ppedziwiatr**

### CODE REVIEW

File: src/gateway/router/routes/sequencerRoute.ts [hunk 1]
```diff
@@ -342,13 +342,12 @@ export async function uploadToBundlr(
...
 
   const bTx = bundlr.createTransaction(JSON.stringify(transaction), { tags });
-  await bTx.sign();
-
-  // TODO: move uploading to a separate Worker, to increase TPS
-  const bundlrResponse = await bTx.upload();
-  logger.debug('Uploading to bundlr', uploadBenchmark.elapsed());
-  logger.debug('Bundlr response id', bundlrResponse.data.id);
-  logger.debug('Bundlr response status', bundlrResponse.status);
+  const bundlrResponse = await bundlr.uploader.uploadTransaction(bTx, { getReceiptSignature: true })
```
Comment (ppedziwiatr) [inline src/gateway/router/routes/sequencerRoute.ts:345]: new API - https://discord.com/channels/@me/936274264397586433/1045729922766876712

**Comment by asiaziola** [comment]
hmm, can you just check `deploBundled` route? I believe we're using old api there...
**    const bundlrResponse = await bundlr.uploader.transactionUploader(dataItem);
**

File: src/gateway/router/routes/deployContractRoute.ts [hunk 2]
```diff
@@ -113,7 +113,7 @@ export async function deployContractRoute(ctx: Router.RouterContext) {
         src_wasm_lang: srcWasmLang || null,
         bundler_src_tx_id: bundlerSrcTxId,
         bundler_src_node: BUNDLR_NODE2_URL,
-        bundler_response: JSON.stringify(srcBundlrResponse.data),
+        bundler_response: JSON.stringify(srcBundlrResponse?.data),
```
Comment (ppedziwiatr) [inline src/gateway/router/routes/deployContractRoute.ts:116]: it doesn't compile without this change:
image

not sure how it worked before...

**Comment by ppedziwiatr** [comment]
> hmm, can you just check `deployBundled` route? I believe we're using old api there...

fixed!

**Pull request merged**